app/main/index.js

```diff
@@ -277,29 +276,18 @@ const handleQueryCommand = (evt, { q: queryPhrase }, plugins) => {
 
 /**
  * Retrieve item details and sends back the response.
- * Loads from cache if necessary.
  *
  * @param {Event} evt
  * @param {Object} item
  */
 const handleItemDetailsRequest = (evt, item) => {
-  // set empty content
-  let content = '';
-  // load from cache if available
-  const configName = `${path.basename(item.plugin.path)}-itemDetails`;
-  const cacheConf = new CacheConf({ configName });
-  const cacheKey = JSON.stringify(item);
-  if (cacheConf.has(cacheKey)) {
-    content = cacheConf.get(cacheKey);
-  } else {
-    // otherwise, load from plugin
-    // eslint-disable-next-line global-require, import/no-dynamic-require
-    const plugin = require(item.plugin.path);
-    content = retrieveItemDetails(item, plugin);
-  }
+  // otherwise, load from plugin
+  // eslint-disable-next-line global-require, import/no-dynamic-require
+  const plugin = require(item.plugin.path);
+  // @todo - cache content
+  const content = retrieveItemDetails(item, plugin);
   // resolve and update the state
   Promise.resolve(content).then(html => {
-    cacheConf.set(cacheKey, html);
```

Comment:
aah cool.
Isn't it a matter of replacing this with `config.set()`, since `conf` actually inherits from `Conf` ? :bowtie: 

maybe that doesn't work and I'm missing something.

Reply:
I thought about it but `conf` in itself never expires whereas in this context, we want the item details content to expire since plugin may be querying an API (github README, etc.)

Reply:
I just removed it for now to keep the function cleaner. I think there's a better way to go about caching without having it too coupled with our fetch functions.